bundle/cluster_manager.go

```diff
@@ -65,3 +65,24 @@ func (b *Bundle) GetClusterManagerClientData(clientType apistructs.ClusterManage
 	}
 	return getResp, nil
 }
+
+func (b *Bundle) ListClusterManagerClientByType(clientType apistructs.ClusterManagerClientType) ([]apistructs.ClusterManagerClientDetail, error) {
```

Comment:
what is ClusterManagerClient

Reply:
![image](https://user-images.githubusercontent.com/30427474/172812179-673ff596-362a-47d3-9a81-f3c372304463.png)

clusterDialerClient -> clusterManagerClient
clusterDialer was uniformly named clusterManager